code/API_definitions/number_verification.yaml

```diff
@@ -0,0 +1,375 @@
+openapi: 3.0.3
+info:
+  title: Number Verification
+  description: |-
+    Service Enabling Network Function API to verify that the provided **phone number** is the one used in the device. It verifies that the user is using a device with the same *phone number* as it is declared.
+    It also make it possible for a Service provider to verify the number itself by returning the phone number retrieved from the authenticated user's access token.
+
+    # Relevant  Definitions and concepts
+    - **NaaS**: *Network-as-a-Service* model where Telco Network resources are exposed to third parties through APIs. In this particular API, Number Verification is exposed following this model. 
```

Comment:
In my opinion this API is more End-User related. NaaS is misleading here.

Reply:
I removed it.

Reply:
thx!